pkg/cluster/bootstrap.go

```diff
@@ -561,6 +561,10 @@ func (c *Cluster) bootstrap(ctx context.Context) error {
 
 	// bootstrap managed database via HTTPS
 	if c.runtime.HTTPBootstrap {
+		// Assuming we should just compare on managed databases
+		if err := c.compareClusterConfig(); err != nil {
```

Comment:
Can this method be renamed to `c.compareConfig` as the c provides the context of "cluster". This is an unnecessary, and less obvious stutter.

Reply:
Yes